### ISSUE TITLE
Dynamically switching to mobile layout removes some UI affordances needed for desktop

### ISSUE THREAD

**Issue by jeremypress** [excalidraw/excalidraw#3515]
Hi,
After the recent change to switch to mobile at small screen sizes, a user who opens excalidraw in a small window will be missing zoom controls that are a big part of the desktop layout. I agree that the zoom buttons aren't needed on mobile, but is there a way users on desktop can still zoom without remembering the keyboard shortcuts?

Maybe instead of switching to mobile we could put the canvas action UI behind a hamburger on desktop like we do for mobile, but keep everything else consistent?

**Comment by thorn0** [comment]
ctrl + mouse wheel

**Comment by jeremypress** [comment]
yep that's definitely a workaround, but I feel like the lib chooses to show zoom on desktop and forward backward on mobile because of the device type, not the screen size. Forward and backward don't do much good since ctrl + z is intuitive, but zooming is less so, which is why there are buttons!

**Comment by dwelle** [comment]
On mobile there's not much space and we're struggling to fit even that which we must. Sure, you can make the argument that larger screens can fit more, but then we'd be having to decide at which breakpoints we need to disable which tools.

**Comment by jeremypress** [comment]
> On mobile there's not much space and we're struggling to fit even that which we must. Sure, you can make the argument that larger screens can fit more, but then we'd be having to decide at which breakpoints we need to disable which tools.

Agreed, I wonder if a better option (but much more work) would be a compact desktop layout based on a breakpoint instead of switching to mobile.

**Comment by dwelle** [comment]
We'd have to change how we _do_ break to mobile.
